index.html

```diff
@@ -1,75 +1,111 @@
-            </div>
-            <div class="text-container">
-                <h2>The Past</h2>
-                <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
-            </div>
-        </section>
-        
-        <section class="middle-content">
-        
-            <h2>Why Did It Have To Be Boxes...</h2>
-            
-            <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
-        
-        </section>
+      <header>
+        <div class="headLinks">
+          <img
+            src="https://github.com/SierraFenstemaker/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/lambda-black.png?raw=true"
+            alt="lambda name"
+          />
+
+          <a href="#">Home</a>
+          <a href="about.html">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </div>
+      </header>
+
+      <section class="top-content">
+        <img
+          src="https://github.com/SierraFenstemaker/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/jumbo.jpg?raw=true"
+          alt="blend"
+        />
+        <div class="text-container">
+          <h2>The Future</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+        <div class="text-container">
+          <h2>The Past</h2>
+          <p>
+            Proin sed quam sed tellus vestibulum ultrices quis in nunc.
+            Phasellus id dui id tortor tincidunt efficitur. Proin faucibus
+            imperdiet erat, non varius lacus. Maecenas non nisl id turpis
+            egestas tincidunt. Nam condimentum venenatis magna eget finibus.
+          </p>
+        </div>
+      </section>
+
+      <section class="middle-content">
+        <h2>Why Did It Have To Be Boxes...</h2>
 
-        <section class="bottom-content">
+        <div class="boxes">
```

Comment:
A good time to use an id is when you are only going to use it one time per page, just like you have done here.